test/Models/esriLineStyleSpec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  getLineStyleCesium,
+  getLineStyleLeaflet
+} from "../../lib/Models/esriLineStyle";
+
+describe("esriLineStyle", function() {
```

Comment:
Description errors -- likely due to copy-and-paste. 
Have been corrected in #4412.

Reply:
we fixed most of the things simultaneous